src/error.rs

```diff
@@ -0,0 +1,25 @@
+//! Contains error types and other error handling tools.
+
+pub use crate::parse::ParseIntError;
+
+/// Impls std::error::Error for the specified type with appropriate attributes, possibly returning
+/// source.
+#[macro_export]
```

Comment:
Do we want to export this? We can make it pub crate only by using the trick that we use in `internal_macros.rs`.

Reply:
Ah, this is the way! Damn! I was confused why the others work and this one not. Macro visibility is confusing me.